app/views/Partners/PartnersEdit.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  StyleSheet,
+  View,
+  TouchableHighlight,
+  FlatList,
+  ActivityIndicator,
+} from 'react-native';
+import {
+  NavigationParams,
+  NavigationScreenProp,
+  NavigationState,
+} from 'react-navigation';
+
+import { Icons } from '../../assets';
+import { NavigationBarWrapper } from '../../components/NavigationBarWrapper';
+import { Typography } from '../../components/Typography';
+import Colors from '../../constants/colors';
+
+import { SvgXml } from 'react-native-svg';
+import { useDispatch, useSelector } from 'react-redux';
+import healthcareAuthorityOptionsSelector from '../../store/selectors/healthcareAuthorityOptionsSelector';
+import getHealthcareAuthoritiesAction from '../../store/actions/healthcareAuthorities/getHealthcareAuthoritiesAction';
+import type { HealthcareAuthority } from '../../store/types';
+import selectedHealthcareAuthoritiesSelector from '../../store/selectors/selectedHealthcareAuthoritiesSelector';
+import toggleSelectedHealthcareAuthorityAction from '../../store/actions/healthcareAuthorities/toggleSelectedHealthcareAuthorityAction';
+
+type PartnersEditScreenProps = {
+  navigation: NavigationScreenProp<NavigationState, NavigationParams>;
+};
+
+const Separator = () => (
+  <View
+    style={{
+      backgroundColor: Colors.DIVIDER,
+      height: StyleSheet.hairlineWidth,
+      width: '100%',
+    }}
+  />
+);
+
+const PartnersScreen = ({
+  navigation,
+}: PartnersEditScreenProps): JSX.Element => {
+  const { t } = useTranslation();
+
+  const dispatch = useDispatch();
+
+  // Fetch list on screen mount
+  useEffect(() => {
+    dispatch(getHealthcareAuthoritiesAction());
+  }, [dispatch]);
+
+  const authorities = useSelector(healthcareAuthorityOptionsSelector);
+  const selectedAuthorities = useSelector(
+    selectedHealthcareAuthoritiesSelector,
```

Comment:
What do you think about inlining these selectors? As a reader of the file you need to have multiple other files open to be able to understand what is happening here.